### DISCUSSION TITLE
US Libre 3.4.0 for Android

### DISCUSSION THREAD

**Discussion by gui-dos** [maheini/FreeStyle-Libre-3-patch#4] [General]
Today an updated release 3.4.0 for the US Play Store has been released and already available i.e. on https://apkpure.com/freestyle-libre-3-–-us/com.freestylelibre3.app.us

**Comment by maheini** [comment]
Thanks for your note and your work on encrypting the FSTL 3 👍
I just added the new version and some notes (free three mod will no longer work with libre 3 version 3.4)
